### PR TITLE
data:alroniks

### PR DESCRIPTION
**Pull request by alroniks**
## Changes proposed

I've added links to my profile.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

Nothing to add

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3554"></a>

### CODE REVIEW

**Review by arpitghura — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

File: data/alroniks.json [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    },
+    {
+      "name": "GoodReads: Readings and reviews",
+      "url": "https://www.goodreads.com/user/show/47526080-ivan-klimchuk",
+      "icon": "FaGoodreads"
+    },
+    {
+      "name": "YouTube: Learn more about software development",
+      "url": "https://www.youtube.com/@klimchukdev",
+      "icon": "FaGoodreads"
```
Comment (kumarsonsoff3) [inline data/alroniks.json:76]: What do you think of using the YouTube icon here!? 🤔

```suggestion
      "icon": "FaYoutube"
```

Comment (alroniks) [reply data/alroniks.json]: Thank you! Nice catch!

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good to merge, Thanks for your Contribution! 🎉

<p align="center"></p>

**Pull request merged**